cloudinit/cmd/devel/logs.py

```diff
@@ -22,11 +24,44 @@
 CLOUDINIT_RUN_DIR = "/run/cloud-init"
 
 
+class ApportFile(NamedTuple):
+    path: str
+    label: str
+
+
+INSTALLER_APPORT_SENSITIVE_FILES = [
```

Comment:
@dbungert please review these potential files and Apport bug labels to see if they make sense or align w/ current expectations for subiquity apport bugs. I've scraped them from latest server live installer and desktop canary  `/usr/share/apport/package-hooks/(subiquity|ubuntu-desktop-installer).py`